server/models/user.go

```diff
@@ -1,8 +1,9 @@
 package models
 
 type User struct {
-	Id       int    `json:"id"`
+	Id       string `json:"id"`
 	Email    string `json:"email"`
 	Username string `json:"username"`
+	Password string `json:"-"`
```

Comment:
```suggestion
	Password string `json:"password,omitempty"`
```

Reply:
This way, if the field doesn't exist, it's ignored

Reply:
@Julius-R will this only hide it if its empty? I want to hide the password in the response for all cases

Reply:
oooh, I gotcha. No, it only hides empty,